Sources/FigSchema/FigSpec.swift

```diff
@@ -158,12 +162,12 @@ public struct FigSpec: Encodable {
             }
         }
 
-        public var names: [String]
-        public var arguments: [Argument]?
+        public var name: [String]
```

Comment:
I wonder whether we'd benefit from an `ExpressibleByArrayLiteral & ExpressibleByStringLiteral` type here so that users could do either of `name = "Foo"` or `name = ["Foo", "Bar"]`

Reply:
Yeah it would be a nice tweak

Reply:
afaik it's possible; you can make a type conform to both aforementioned protocols and it should work out of the box.

Reply:
I'll try it!